apps/sms/test/marionette/conversion_banner_test.js

```diff
@@ -145,11 +143,11 @@ marionette('Message Type Conversion Banner', function() {
     test('The banner is not shown after sending another message',
     function() {
       navigateToMMSConversation();
-      messagesApp.Composer.messageInput.sendKeys('Another message');
+      composer.messageInput.sendKeys('Another message');
```

Comment:
No :) We have ```NewMessageView.typeMessage```, but it's too much work to migrate to "views" in every tests in one go, that is why you see new tests with "views" and old tests with "messageApp.old-property-that-gives-direct-access-to-accessors" :)